src/Type/Php/DateTimeDynamicReturnTypeExtension.php

```diff
@@ -14,34 +14,34 @@
 use PHPStan\Type\ObjectType;
 use PHPStan\Type\Type;
 use function count;
+use function date_create;
 use function in_array;
 
 class DateTimeDynamicReturnTypeExtension implements DynamicFunctionReturnTypeExtension
 {
 
 	public function isFunctionSupported(FunctionReflection $functionReflection): bool
 	{
-		return in_array($functionReflection->getName(), ['date_create_from_format', 'date_create_immutable_from_format'], true);
+		return in_array($functionReflection->getName(), ['date_create', 'date_create_immutable'], true);
```

Comment:
I can't read this diff. Please give a different name to your new extension and don't change an existing file.

Reply:
Diff should be better now.